tests/Integration/Common/IntegrationTestCase.php

```diff
@@ -33,4 +33,15 @@ public function assertSpans($traces, $expectedSpans)
     {
         $this->assertExpectedSpans($this, $traces, $expectedSpans);
     }
+
+    /**
+     * Checks that the provide span exists in the provided traces and matches expectations.
+     *
+     * @param $traces
```

Comment:
Not sure if this is the correct type declaration here, but I think it should be:
```suggestion
     * @param Span[][] $traces
```